chapters/annotations.tex

```diff
@@ -42,6 +42,7 @@ \section{Documentation}\label{annotations-for-documentation}\label{documentation
   String info = "" "Description of the class";
   String revisions = "" "Revision history";
   Figure[:] figures = {}; "Simulation result figures";
+  String styleSheet = "" "Only at top, and alternatively String[:]";
```

Comment:
This must be the first time we try to do this with the record style.  Wouldn't it be better to have the array as the main case, as it allows for a better default? Also, the description string should focus on purpose rather than syntax and restrictions.
```suggestion
  String[:] styleSheets = {} "Style sheets for documentation and revision history";
```
(A shorter description string might fit the format of the specification document better.)

Reply:
Here we see the problem of describing info as Documentation; without that change we can just use 

`String[:] styleSheets = {} "Style sheets for documentation";`

Where documentation refer to the Documentation-node.

Reply:
I will add that change.

Reply:
Thanks.  Resolving.